### PR TITLE
PAI Job metrics view

### PR DESCRIPTION
**Pull request by YanjieGao**
PAI Job metrics view, user could view job performance from 3 levels. It contains:

- JobLevelMetrics
- TaskRoleLevelMetrics
- TaskLevelMetrics

For example:
JobLevelMetrics
image

TaskRoleLevelMetrics
image

TaskLevelMetrics
image

### CODE REVIEW

**Comment by fanyangCS** [comment]
> #!/bin/bash

do you want to remove this file?

---
Refers to: service-deployment/bootstrap/grafana/configmap-create.sh:1 in 2108fb6. [](commit_id = 2108fb63362f162f3c8cb0625b8c328b71ea9190, deletion_comment = True)

File: service-deployment/service.yaml [hunk 1]
```diff
@@ -214,3 +217,4 @@ servicelist:
       - cleanup.sh
     startscript: start.sh
     stopscript: cleanup.sh
+#
```
Comment (fanyangCS) [inline service-deployment/service.yaml]: why do we need this?

**Comment by YanjieGao** [comment]
nochanges to file service-deployment/bootstrap/grafana/configmap-create.sh

**Comment by fanyangCS** [comment]
I can still see it in the PR, please exclude it in this PR

---
In reply to: [375172066](https://github.com/Microsoft/pai/pull/387#issuecomment-375172066) [](ancestors = 375172066)

**Comment by YanjieGao** [comment]
removed

**Pull request merged**